data/ShailavMalik.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Shailav Malik",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Coding Enthusiast | Frontend development",
+    "avatar": "https://github.com/ShailavMalik.png",
```

Comment:
@ShailavMalik remove the avatar it will be taken from your GitHub

Reply:
OK

Reply:
Made the changes